src/styles/blocks/card.css

```diff
@@ -0,0 +1,70 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  box-sizing: content-box;
+  height: 134px;
+  width: 160px;
+  padding: 0 3px;
+  object-fit: contain;
+}
+
+.card__name-of-product {
+  width: 166px;
+  padding-top: 38px;
```

Comment:
As for me it's more logic to set margins to set distance between elements, I mean paddings - increases element size, so if there no need to increase it (for example clickable zone), more logic to use margins. But it's not critical, just a suggestion

Reply:
Thank you, I will take all advice into account in the future)